protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/compensation/CompensationSubscriptionRecord.java

```diff
@@ -145,6 +154,12 @@ public CompensationSubscriptionRecord setCompensableActivityId(
     return this;
   }
 
+  public CompensationSubscriptionRecord setCompensationActivityElementId(
+      final String compensationActivityElementId) {
+    compensationActivityElementIdProperty.setValue(compensationActivityElementId);
+    return this;
+  }
```

Comment:
:wrench: We should align the name of the setter, the property, and the property name to `compensationHandlerId`.

Only the getter has been renamed so far. :smile:

Reply:
You are right 🤣 🙇